unit_tests/kernels/UnitTestKernelUtils.C

```diff
@@ -956,702 +955,6 @@ calc_exposed_area_vec(
   areaVec.sync_to_host();
 }
 
-#if !defined(KOKKOS_ENABLE_GPU)
```

Comment:
@overfelt I trust you on this, but I'm also curious as to why are we removing all these tests?

Reply:
I started to refactor all of these functions when I noticed the `#if 0` disabling a couple of functions.  After deleting all of that code, I saw that what was left was not called from any other function that I could find, so I just deleted that also.  These are not tests, but utility functions and I figured if the unit tests compiled, linked and ran without the these utilities, we could remove it.